### ISSUE TITLE
Fine-tune T5 for Text2SQL

### ISSUE THREAD

**Issue by drustanyjt** [drustanyjt/t5-for-sparql#10]
To test inferencing, we will need to re-implement the training functions done by Tscholak and Shaw.

**Comment by drustanyjt** [comment]
A few things need to be done first to get this up and running

- [ ] #3 
- [ ] #11 
- [ ] Speeding up training with [Accelerate](https://huggingface.co/docs/transformers/accelerate)

**Comment by drustanyjt** [comment]
I think there was a misunderstanding of what is a metric. The eval metric is independent of the optimizing during training. Optimization is still done using backprop and grad descent.
